arm-hdinsight/2015-03-01-preview/examples/ChangeHttpConnectivityEnable.json

```diff
@@ -0,0 +1,24 @@
+{
+  "parameters": {
+    "name": "cluster1",
+    "resourceGroupName": "rg1",
+    "api-version": "2015-03-01-preview",
+    "subscriptionId": "subid",
+    "httpSettingsParameters": {
+      "restAuthCredential.isEnabled": true,
+      "restAuthCredential.username": "hadoop",
+      "restAuthCredential.password": "Password"

```

Comment:
Throughout, for other examples, we use ********** instead of fake password strings. It's more obvious and forces the user to enter something new. Let's stick with this: "**********"